poc_iot_injector/src/server.rs

```diff
@@ -206,10 +196,6 @@ async fn process_submission(
                         tracing::warn!("unable to submit_txn, {err:?}")
                     }
                 }
-            } else {
```

Comment:
don't we still want this else block to just report the metrics and tracing calls while we still have the `do_submission` boolean? conversely does it make sense to just get rid of do_submission now and from here on assume if you're running the injector at all you want it to attempt to submit to the chain?

Reply:
Updated [46976f9](https://github.com/helium/oracles/pull/278/commits/46976f93de969d23dc3699f92a6625fa7a5b33a4)